Project-Euler/Project-Euler/Problems.cs

```diff
@@ -105,6 +103,46 @@ private static bool IsItPrime(long number)
             }
 
             return true;
-        }  
+        }
+
+        public static void Problem4()
+        {
+            Console.WriteLine("\n\t1)Find the largest palindrome made from the product of two 3 digit numbers.");
+
+            int currentProduct;
+            int greatestProduct = 0;
+
+            // This loops iterates through every single possible combination of 3-digit numbers and multiplays them. 
+            for (int i = 999; i > 99; i--)
+            {
+                for (int j = i; j > 99; j--)
+                {
+                    currentProduct = i * j;
+
+                    if (IsItPalendromic(currentProduct) && currentProduct > greatestProduct)
```

Comment:
It's fine if you want to leave it. But the convention in CS would be to name this IsPalendrome(). 'It' is never used in bool function names.

Examples of convention bool function names:

- IsEmpty (not IsItEmpty)
- IsValid (not IsItValid)
- HasErrors (not ItHasErrors)